src/pkg/bundle/bundle.go

```diff
@@ -110,6 +141,11 @@ func Create(b *Bundler, signature []byte) error {
 
 			zarfPkgDesc, err := localBundler.ToBundle(store, zarfPkg, artifactPathMap, b.tmp, pkgTmp)
 
+			// add package name annotations, for local zarf packages, these names will be the same
```

Comment:
What do you think of making a follow-up ticket for handling UDS pkg name vs Zarf pkg name for local bundle?